vnext/Shared/DevSettings.h

```diff
@@ -30,9 +30,10 @@ enum class JSIEngineOverride : int32_t {
   ChakraCore = 2, // Use the JSIExecutorFactory with ChakraCoreRuntime
   Hermes = 3, // Use the JSIExecutorFactory with Hermes
   V8 = 4, // Use the JSIExecutorFactory with V8
-  V8NAPI = 5, // Use the JSIExecutorFactory with V8-NAPI
+  V8Lite = 5, // Use the JSIExecutorFactory with V8-NAPI
```

Comment:
>V8Lite [](http://example.com/codeflow?start=2&length=6)

Why is it needed?

Reply:
I do not know, but it exists both here and in the Office code base.
Keeping it to make the change less disruptive.